### PR TITLE
Updated files (incl. Error + Admin .html

### PR DESCRIPTION
**Pull request by hazzakak**

### CODE REVIEW

**Comment by hazzakak** [comment]
Added correct variable syntax

File: static/styles/style.css [hunk 1]
```diff
@@ -1,25 +1,22 @@
 @font-face {
   font-family: "pricedown";
   src: url("../fonts/pricedown.eot");
-  src:
-  url("../fonts/pricedown.woff") format("woff"),
-  url("../fonts/pricedown.otf") format("opentype"),
-  url("../fonts/pricedown.svg#filename") format("svg");
+  src: url("fonts/pricedown.woff") format("woff"), url("../fonts/pricedown.otf") format("opentype"), url("../fonts/pricedown.svg#filename") format("svg");
```
Comment (Aareon) [inline static/styles/style.css]: missing `../` in `url("../fonts/pricedown.woff")`

File: templates/index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h1>SourceCAD</h1>
   <!--<img class="image-top" src="image/banner.png" alt=""> -->
```
Comment (Aareon) [inline templates/index.html]: Title should be `Computer Aided Dispatch`, not `Computer-aided design`. This goes for all files.

File: templates/admin.html [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+          <tr>
+            <th>User</th>
+            <th>Password</th>
+
+            <th>Role</th>
+            <th>User Snowflake</th>
+          </tr>
+          <!-- in onclick="%edit%" change the hash to a edit portion -->
+          <tr onclick="#">
+            <td>Harry</td>
```
Comment (Aareon) [inline templates/admin.html]: Please don't hard-code values like these.

File: templates/admin.html [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+          <caption>User Activity</caption>
+          <tr>
+            <th>User</th>
+            <th>Role</th>
+            <th>Last Login</th>
+            <th>Last Login Time</th>
+          </tr>
+          <!-- in onclick="%edit%" change the hash to a edit portion -->
+          <tr onclick="#">
+            <td>Harry (1B-03)</td>
```
Comment (Aareon) [inline templates/admin.html]: And here

File: templates/dispatch.html [hunk 5]
```diff
@@ -25,119 +25,62 @@ <h1>SourceCAD</h1>
...
-          </div>
-          <div class="dispatch-callout-current">
-            Reason: {reason} <br>
-            Location: {location} <br>
-            Details: {details}<br>
-            On-scene Units: {units} <br>
-          </div>
+      <div class="dispatch-calls">
+        <div class="dispatch-callout-current">
+          Reason: {reason} <br> Location: {location} <br> Details: {details}<br> On-scene Units: {units} <br>
```
Comment (Aareon) [inline templates/dispatch.html]: Needs proper Flask syntax. i.e. `{{reason}}`

**Review by Aareon — CHANGES_REQUESTED** [review]
Please correct these so that I can merge.

**Review by Aareon — APPROVED** [review]

File: templates/admin.html [hunk 6]
```diff
@@ -55,184 +55,10 @@
...
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
+            <td>{{user} (1B-03)</td>
```
Comment (Aareon) [inline templates/admin.html]: Think you missed something here. I'll end up generating this table myself, so really just leave one column with the variables and I'll handle the rest.

**Review by Aareon — CHANGES_REQUESTED** [review]

File: templates/admin.html [hunk 7]
```diff
@@ -55,7 +55,7 @@
           </tr>
           <!-- in onclick="%edit%" change the hash to a edit portion -->
           <tr onclick="#">
-            <td>{{user} (1B-03)</td>
+            <td>{{username}} (1B-03)</td>
```
Comment (Aareon) [inline templates/admin.html]: Left your badge number.

**Review by Aareon — CHANGES_REQUESTED** [review]

**Comment by Aareon** [comment]
Please alter this PR to fit the latest commits.

**Review by Aareon — APPROVED** [review]

**Pull request merged**